addons/account/models/account_move.py

```diff
@@ -872,26 +873,24 @@ def _compute_name(self):
                 continue
 
             move_has_name = move.name and move.name != '/'
-            if move_has_name or move.state != 'posted':
-                if not move.posted_before and not move._sequence_matches_date():
-                    if move._get_last_sequence():
-                        # The name does not match the date and the move is not the first in the period:
-                        # Reset to draft
-                        move.name = False
-                        continue
-                else:
-                    if move_has_name and move.posted_before or not move_has_name and move._get_last_sequence():
-                        # The move either
-                        # - has a name and was posted before, or
-                        # - doesn't have a name, but is not the first in the period
-                        # so we don't recompute the name
-                        continue
-            if move.date and (not move_has_name or not move._sequence_matches_date()):
+            if move_has_name or move.state == 'draft':
+                if move_has_name and not move.posted_before and not move._sequence_matches_date():
+                    # The name does not match the date and the move is not the first in the period:
+                    # Reset to draft
+                    move.name = False
+                    continue
+            if move.date and not move_has_name and move.state != 'draft':
                 move._set_next_sequence()
 
-        self.filtered(lambda m: not m.name and not move.quick_edit_mode).name = '/'
         self._inverse_name()
 
+    @api.depends('date', 'journal_id', 'move_type', 'name', 'posted_before', 'sequence_number', 'sequence_prefix', 'state')
+    def _compute_name_placeholder_form(self):
+        self.name_placeholder_form = False
+        for record in self.filtered(lambda move: (not move.name or not move.posted_before) and (not move._get_last_sequence() or move.state == "posted")):
+            sequence_format_string, sequence_format_values = record._get_sequence_format_param(record._get_starting_sequence())
+            sequence_format_values['seq'] = sequence_format_values['seq'] + 1
+            record.name_placeholder_form = sequence_format_string.format(**sequence_format_values)
```

Comment:
I didn't understand your previous condition, did I make a mistake here?
```suggestion
        for move in self:
            if (
                not move.name
                and not move.posted_before
                and not move._get_last_sequence()
            ):
                sequence_format_string, sequence_format_values = move._get_sequence_format_param(move._get_starting_sequence())
                sequence_format_values['seq'] = sequence_format_values['seq'] + 1
                move.name_placeholder_form = sequence_format_string.format(**sequence_format_values)
            else:
                movename_placeholder_form = False
```